StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OpenAuthenticatedWebViewMessageHandler.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  OpenAuthenticatedWebViewMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/14/24.
+//
+
+import Foundation
+
+// Indicates to open the provided URL in a secure webview. 
```

Comment:
Sorry, I missed these references when I gave my comment suggestion
```suggestion
// Indicates to open the provided URL in an ASWebAuthenticationSession. 
```